devops/pypi-release.yml

```diff
@@ -33,48 +33,50 @@ stages:
     vmImage: $(PoolImage)
     
   jobs:
-  - template: all-tests-job-template.yml
+  - template: templates/all-tests-job-template.yml
     parameters:
       name: Linux
       vmImage: 'ubuntu-16.04'
       pyVersions: [3.5, 3.6, 3.7]
       freezeArtifactStem: "PreDeployment"
       freezeFileStem: $(FreezeFileStem)
 
-  - template: all-tests-job-template.yml
+  - template: templates/all-tests-job-template.yml
     parameters:
       name: Windows
       vmImage:  'vs2017-win2016'
       pyVersions: [3.5, 3.6, 3.7]
       freezeArtifactStem: "PreDeployment"
       freezeFileStem: $(FreezeFileStem)
       
-  - template: all-tests-job-template.yml
+  - template: templates/all-tests-job-template.yml
     parameters:
       name: MacOS
       vmImage:  'macos-latest'
       pyVersions: [3.6, 3.7] 
```

Comment:
Can we add 3.5 back in here, and run tests without matplotlib?

Reply:
I was going to do it afterwards but you're right, why not...

Reply:
note that according to NEP18 (I think) py3.5 is no longer supported by the pydata ecosystem. matplotlib and the upcoming pandas release have already dropped it, and sklearn is dropping it for the next release.